### ISSUE TITLE
Keyboard Layout not persistent on reboot

### ISSUE THREAD

**Issue by Riggs9715** [pop-os/cosmic-settings-daemon#42]
When adding UK keyboard layout in settings and removing the US layout, on reboot the setting reverts. The new layout is removed and US is restored in its place. Setting must therefore be changed every boot.

**Comment by Riggs9715** [comment]
Yes, that's working as a workaround now to set the correct layout, so it seems its just the settings ability to change it thats not working. This is on arch using the AUR cosmic-session-git package.

**Comment by Riggs9715** [comment]
I just tried to check for the polkit files you mentioned as well.  /usr/share/polkit-1/rules.d contains cosmic-settings.rules and cosmic-settings-daemon.rules

**Comment by Riggs9715** [comment]
Update. I believe I have found both the cause and the fix. cosmic-settings-daemon.rules contains an if statement for
subject.isInGroup("sudo")
Whereas on arch wheel group is more typical for giving a user sudo access. I changed sudo to wheel, rebooted, changed the layout through the settings, rebooted again and this time it stayed changed. 
Could the polkit files be updated to allow for wheel group users as well as sudo group?
